src/main/java/ninja/amp/ampmenus/menus/ItemMenu.java

```diff
@@ -127,7 +128,20 @@ public void setParent(ItemMenu parent) {
      * @param menuItem The {@link ninja.amp.ampmenus.items.MenuItem}.
      * @return The {@link ninja.amp.ampmenus.menus.ItemMenu}.
      */
+    @Deprecated
     public ItemMenu setItem(int position, MenuItem menuItem) {
```

Comment:
Method isn't redundant. If you have compiled code before update, you cannot change method signature.
I can remove it, but all devs need to recompile their plugins.